arrows/core/derive_metadata.cxx

```diff
@@ -77,6 +84,13 @@ get_sensor_rotation( kwiver::vital::metadata_sptr const& metadata )
   auto const pitch = pitch_item.as_double() * kv::deg_to_rad;
   auto const roll = roll_item.as_double() * kv::deg_to_rad;
 
+  if( yaw < 0 || yaw > 360 * kv::deg_to_rad || pitch < -180 * kv::deg_to_rad ||
+      pitch > 180 * kv::deg_to_rad || roll < 0 || roll > 360 * kv::deg_to_rad )
```

Comment:
Shouldn't roll be ±180°? For that matter, are we sure yaw is always normalized to 0-360°?

Reply:
This is all taken from the reference implementation. Unless we know that we've changed a convention, it seems reasonable to just ensure parity for now.